### PR TITLE
chore: update react-native

### PR DESCRIPTION
**Pull request by shadowsheep1**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/languages-and-frameworks/react-native.md
Comment (balanza) [inline radar/languages-and-frameworks/react-native.md:20]: **issue:** ‏I find little value in having these kinds of considerations - they seem more like a sale pitch from Meta. I'd rather point out what are the **proven** benefits for us.

**Review by balanza — CHANGES_REQUESTED** [review]
I mostly argue about having a description that could apply to any company. Being a tool for **our** decisions, I expect we highlight  the features that make the difference for us.

File: radar/languages-and-frameworks/react-native.md
Comment (shadowsheep1) [reply radar/languages-and-frameworks/react-native.md:20]: If we have to write only our benefits I agree with you, if we have to extract also some key feature of the tool/language also for the people reading to let them have more insights, maybe not. But It depends on how we should write it.  

Should it be better to point our "best practice" on what to write on these section on the Readme?

File: radar/languages-and-frameworks/react-native.md
Comment (shadowsheep1) [reply radar/languages-and-frameworks/react-native.md:24]: As below. This is a general use case. I didn't get that I need to write only our specific use case.

Comment (gunzip) [reply radar/languages-and-frameworks/react-native.md:24]: This section here is to list how we (should) use this technology. I'd rephrase this into something like:
"Use React Native whenever you're building a mobile app both for iOS and Android"

**Review by gunzip — CHANGES_REQUESTED** [review]

**Comment by gunzip** [comment]
What about expo? Let's say we must build a new mobile app, do you think it's worth to have a new entry for expo? (assess?)

File: radar/languages-and-frameworks/react-native.md
Comment (shadowsheep1) [reply radar/languages-and-frameworks/react-native.md]: I simplified the whole card.

**Comment by shadowsheep1** [comment]
> What about expo? Let's say we must build a new mobile app, do you think it's worth to have a new entry for expo? (assess?)
Why not? We could consider it for sure.

**Review by gunzip — APPROVED** [review]

**Comment by shadowsheep1** [comment]
@balanza check it out now. We're waiting for your approval.

**Review by balanza — APPROVED** [review]

**Pull request merged**
